apps/etterlatte-vilkaarsvurdering/src/main/kotlin/vilkaarsvurdering/VilkaarsvurderingRoutes.kt

```diff
@@ -15,29 +15,24 @@ import no.nav.etterlatte.libs.common.vilkaarsvurdering.VilkaarVurderingData
 import no.nav.etterlatte.libs.common.vilkaarsvurdering.VilkaarsvurderingResultat
 import no.nav.etterlatte.libs.common.vilkaarsvurdering.VilkaarsvurderingUtfall
 import no.nav.etterlatte.libs.common.withBehandlingId
+import no.nav.etterlatte.libs.common.withParam
 import no.nav.etterlatte.libs.ktor.accesstoken
 import no.nav.etterlatte.libs.ktor.saksbehandler
 import java.time.LocalDateTime
 import java.util.*
 
-fun Route.vilkaarsvurdering(
-    vilkaarsvurderingService: VilkaarsvurderingService
-) {
+fun Route.vilkaarsvurdering(vilkaarsvurderingService: VilkaarsvurderingService) {
     route("/api/vilkaarsvurdering") {
         val logger = application.log
 
         get("/{behandlingId}") {
             withBehandlingId { behandlingId ->
-                logger.info("Henter vilkårsvurdering for $behandlingId")
-
                 try {
-                    val vilkaarsvurdering = vilkaarsvurderingService.hentEllerOpprettVilkaarsvurdering(
-                        behandlingId = behandlingId,
-                        accessToken = accesstoken
-                    )
-                    call.respond(vilkaarsvurdering.toDto())
+                    logger.info("Henter vilkårsvurdering for $behandlingId")
+                    vilkaarsvurderingService.hentEllerOpprettVilkaarsvurdering(behandlingId, accesstoken)
+                        .let { call.respond(it.toDto()) }
```

Comment:
Ikke veldig sterk mening, men _liker_ bedre å unngå let i sånne caser. Leser raskere hva som skjer i den forrige syntaksen liksom, mens jeg her må "lete" etter hva som skjer

Reply:
Handler det litt om scope og forventning fra en service? Altså at det som returneres fra en service ikke skal være ansvarlig for å lage en `response` ?

Reply:
Gjør det om slik det var før - er litt vant til å bruke `.let` på den slags selv, men om det gjør det vanskeligere å lese så er jeg helt for å gjøre det så enkelt som mulig 👍